src/analytics/src/main/scala/osmesa/analytics/oneoffs/ChangesetStatsCreator.scala

```diff
@@ -155,6 +178,16 @@ object ChangesetStatsCreator
                 }
               })
 
+            // Distributing these writes to the executors to avoid no suitable driver errors on master node
```

Comment:
_shakes fist at JDBC!!!_

Does invoking the name of the driver help? This is a weird workaround to need to remember to do.

Reply:
I agree that it's weird.  But it appears to be a classpath thing.  Works fine locally, where everything is installed in one place.  Something about the setup on YARN is making this necessary.  Found no pointers of any use on the world wide web.

Reply:
_le sigh_